src/open_samus_returns_rando/specific_patches/static_fixes.py

```diff
@@ -320,7 +320,7 @@ def disable_vignettes(editor: PatcherEditor) -> None:
         ],
         # Middle Save Station Water Bomb Blocks
         "s090_area9": [
-            {"block_group": 66, "cc": ["007"], "vignette_models": [3224282959]}
+            {"block_group": 66, "cc": ["007"], "vignette_models": [3224282959, 1]}
```

Comment:
Somehow I thought that this is the result of a CRC. Is this really just `1`?

Reply:
Yeah it's part of a separate group in that sg